cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,9 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and image header size for bootloader support
+export IMAGE_ALIGN = 8
+export IMAGE_HDR_SIZE = 512
```

Comment:
I wonder if these values are specific to the CPU (and if, I guess they should go into `nrf5x_common`...), or if they are the same for all cortex-m base CPUs?